client/fileInput.jsx

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+import { assertTSPropertySignature } from 'babel-types';
+import FormData from 'form-data'
+let data = new FormData();
```

Comment:
seems you don't use this.

Reply:
I believe the only one I don't use is
`let data = new FormData();`